contributing.md

```diff
@@ -87,6 +87,16 @@ and ideally reproduce the bug. Simply encountering an error does not mean a bug
 reported; as below, search JIRA and search and inquire on the Spark user / dev mailing lists 
 first. Unreproducible bugs, or simple error reports, may be closed.
 
+It's very helpful if the bug report has a description about how the bug was introduced, by 
+which commit, so that reviewers can easily understand the bug. It also helps committers to 
```

Comment:
I would add a self-reproducer too ..

Reply:
It's already mentioned in the paragraph above
> Bug reports are only useful however if they include enough information to understand, isolate 
 and ideally reproduce the bug.

Reply:
How about additionally mentioning narrowing down the problem? At least I spend some time to ask the information and narrow down the issue time to time.